src/main/scala/org/constellation/ConstellationNode.scala

```diff
@@ -81,14 +82,28 @@ object ConstellationNode extends IOApp {
       fromArg.getOrElse(Option(System.getenv(env)).map(_.toInt).getOrElse(config.getInt(configPath)))
     }
 
+  private def getKeyPair[F[_]: Sync](cliConfig: CliConfig): F[KeyPair] =
+    if (cliConfig.keyStorePath != null) {
+      KeyStoreUtils
+        .fromPath(
+          cliConfig.keyStorePath,
+          cliConfig.alias,
+          cliConfig.storePassword.toCharArray,
+          cliConfig.keyPassword.toCharArray
+        )
+        .use(_.pure[F])
+      // TODO: kpudlik: Fallback. Remove after forcing node operators to use keystore only.
```

Comment:
It would be nice to have a task for preparing `terraform/gcp` and as one of DoD would be removing the fallback

Reply:
Done 👍 

https://github.com/Constellation-Labs/constellation/issues/638